crates/runtime_crypto/src/methods/encrypt.rs

```diff
@@ -35,6 +40,22 @@ pub fn encrypt(algorithm: Algorithm, key_value: Vec<u8>, data: Vec<u8>) -> Resul
                 "invalid counter length. Currently supported 32/64/128 bits",
             )),
         },
+        Algorithm::RsaOaep(label) => {
+            let public_key = RsaPrivateKey::from_pkcs1_der(&key_value)?.to_public_key();
+            let mut rng = OsRng;
+            let padding = match label {
```

Comment:
@QuiiBz  if label isn't an Uint8Array, l will use Oaep::new::<Sha256>().

Reply:
Yes, but that means we allow a `label` that can be anything which fallbacks to none, instead of throwing an error to the user